From 7337bd1fba49e71c2a87d7261bcebbd356e1be50 Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@xxxxx.com>
Date: Sun, 4 Feb 2024 09:21:01 -0500
Subject: [PATCH] drivers: net: wireless: broadcom: brcm80211: brcmfmac: core

brcmf_inetaddr_changed: fail to get arp ip table err:-52

The firmware used by the unit is from 2013. I suspect there
will be no available updates for it. So lets hide the error
msg.

Tested-on: H96-MAX X3
Signed-off-by: Patrick Yavitz <pyavitz@xxxxx.com>
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/core.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/core.c
index 175272c2694d..3a1f98aa871b 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/core.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/core.c
@@ -1020,7 +1020,7 @@ static int brcmf_inetaddr_changed(struct notifier_block *nb,
 	ret = brcmf_fil_iovar_data_get(ifp, "arp_hostip", addr_table,
 				       sizeof(addr_table));
 	if (ret) {
-		bphy_err(drvr, "fail to get arp ip table err:%d\n", ret);
+		//bphy_err(drvr, "fail to get arp ip table err:%d\n", ret);
 		return NOTIFY_OK;
 	}
 
-- 
2.39.2

